testutils/test_data.go

```diff
@@ -0,0 +1,31 @@
+package testutils
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+)
+
+// Test data for generic loading of JSON data files.
+type TestData[T any] struct {
+	Input T `json:"input"`
+}
+
+// Create a new instance of `TestData` with the given input data.
+func NewTestData[T any](defaultInput T) TestData[T] {
```

Comment:
is there any test file we should add to test this?

Reply:
The idea is to run these compliance tests from a separate framework, the test files will be set in [that repo](https://github.com/Layr-Labs/eigensdk-compliance).

Reply:
oh interesting - make ssense